builtin/mainmenu/dlg_reinstall_mtg.lua

```diff
@@ -0,0 +1,114 @@
+--Minetest
+--Copyright (C) 2023 Gregor Parzefall
+--
+--This program is free software; you can redistribute it and/or modify
+--it under the terms of the GNU Lesser General Public License as published by
+--the Free Software Foundation; either version 2.1 of the License, or
+--(at your option) any later version.
+--
+--This program is distributed in the hope that it will be useful,
+--but WITHOUT ANY WARRANTY; without even the implied warranty of
+--MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+--GNU Lesser General Public License for more details.
+--
+--You should have received a copy of the GNU Lesser General Public License along
+--with this program; if not, write to the Free Software Foundation, Inc.,
+--51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+
+function check_reinstall_mtg()
+	if core.settings:get_bool("no_mtg_notification") then
```

Comment:
I believe we still document internal settings in settingtypes.txt

Reply:
Done.